packages/directory/index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  name: 'navi-directory'
+  name: require('./package').name
```

Comment:
what's going on here, did a codemod do this?  Does this really resolve to the same thing?

Reply:
nm I see that it would end up being the same thing.